src/main/java/org/fundacionjala/sfdc/pageobjects/contacts/SFCDetailsPage.java

```diff
@@ -35,6 +36,11 @@ public SFCDetailsPage() {
         PageFactory.initElements(this.webDriver, this);
     }
 
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)